modules/cache.js

```diff
@@ -165,11 +165,13 @@ Cache.prototype.onNewBlock = function (block, broadcast, cb) {
 };
 
 /**
- * This function will be triggered when a round finishes, it will clear all cache entires.
- * @param {Round} round
+ *This function will be triggered when round has changed, it will clear all cache entires.
```

Comment:
`* This function will be triggered when round has changed, it will clear all cache entires.`